### DISCUSSION TITLE
Time_scale in HPC Vs. Local system

### DISCUSSION THREAD

**Discussion by ashishdhole** [idaholab/moose#27519] [Q&A Modules: Phase field]
Hello,
If I am running same input script on my local PC the 1st time step converges well at 1 sec, but if I am running the same script on HPC, my 1st time step converges at 6.25e-05. is there anything I need to add in the script that will show same time steps on HPC as on my local system?
Thank you

**Comment by GiudGiud** [comment]
hello

Are you using the same number of cores on both systems?

Guillaume

**Reply by ashishdhole** [reply]
no. On my local system I am using 4 cores on HPC I am using 400.

**Reply by GiudGiud** [reply]
the preconditioning depends on the number of cores used.
We recommend 20k dofs per core as a guideline. 400 processes for an input that runs with 4 is likely overkill and detrimental to the runtime

**Reply by ashishdhole** [reply]
In case of adaptivity how will we know the exact dof?

**Reply by GiudGiud** [reply]
you can use a postprocessor to count the number of dofs
https://mooseframework.inl.gov/source/postprocessors/NumDOFs.html

uniform refinement generates 4x (in 2D) and 8x (in 3D) as many cells for the refined cells
